dask/bag/core.py

```diff
@@ -458,7 +455,7 @@ def distinct(self):
 
         return type(self)(merge(self.dask, dsk, dsk2), b, 1)
 
-    def reduction(self, perpartition, aggregate):
+    def reduction(self, perpartition, aggregate, split_every=None, out=Item):
```

Comment:
Fixed

On Thu, Feb 11, 2016 at 1:45 PM, Jim Crist <notifications@github.com> wrote:

> In dask/bag/core.py
> <https://github.com/dask/dask/pull/980#discussion_r52672169>:
>
> > @@ -458,7 +455,7 @@ def distinct(self):
> >
> >          return type(self)(merge(self.dask, dsk, dsk2), b, 1)
> >
> > -    def reduction(self, perpartition, aggregate):
> > +    def reduction(self, perpartition, aggregate, split_every=None, out=Item):
>
> Might want to document out. Perhaps out_type as well, just to be a bit
> more descriptive? Otherwise, this looks good.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dask/dask/pull/980/files#r52672169>.
>